test/acceptance/features/bindAppToServiceAnnotations.feature

```diff
@@ -542,3 +542,49 @@ Feature: Bind an application to a service using annotations
             """
         Then Service Binding "binding-app-via-gvk" is ready
         And The application env var "BACKEND_HOST_INTERNAL_DB" has value "internal.db.stable.example.com"
+
+    Scenario: Bind application to service with invalid sourceValue annotations
```

Comment:
the scenario description suggests that the binding is performed, but actually it is not. Hence, we should reflect that, e.g.

"Application cannot be bound to service containing annotation with an invalid sourceValue value"

Reply:
Fixed in 6054ca0c.